pep-0458.txt

```diff
@@ -154,41 +154,42 @@ This PEP focuses only on integrating TUF into PyPI. However, the reader is
 encouraged to review TUF design principles [2]_ and SHOULD be
 familiar with the TUF specification [16]_.
 
-Terms used in this PEP are defined as follows:
+The following terms used in this PEP are defined in the Python Packaging
+Glossary [17]_: *project*, *release*, *distribution*.
 
-* Projects: Projects are software components that are made available for
-  integration.  Projects include Python libraries, frameworks, scripts,
-  plugins, applications, collections of data or other resources, and various
-  combinations thereof.  Public Python projects are typically registered on the
-  Python Package Index [17]_.
+Additional terms used in this PEP are defined as follows:
 
-* Releases: Releases are uniquely identified snapshots of a project [17]_.
+* Role: TUF specifies one *root* role and multiple other roles to which the
+  *root* role delegates responsibilities, directly or indirectly. The term
+  *top-level* role refers to the *root* role and any role specified directly by
+  the *root* role, i.e. *timestamp*, *snapshot* and *targets* roles. Each role
+  has a single metadata file that it is trusted to provide.
 
-* Distributions: Distributions are the packaged files that are used to publish
-  and distribute a release [17]_.
+* Distribution file: A versioned archive file that contains Python packages,
+  modules, and other resource files that are used to distribute a release. In
+  this document a distribution file may also be referred to with the single
+  word distribution.
```

Comment:
Worth pointing out that this is equivalent to the term "Distribution Package" in the Glossary of the Python Packaging User Guide?

Reply:
I briefly considered that. But it seemed more confusing than helpful.

An alternative would be to keep using the words "distribution" and "package" as before and adopt PyPI's definition of "distribution package", which also says that both words can be used alone to say the same thing.